betty/tests/coverage/test_coverage.py

```diff
@@ -509,14 +509,14 @@ class TestKnownToBeMissing:
         "UserFacingEntity": TestKnownToBeMissing,
     },
     "betty/model/association.py": {
-        "BidirectionalEntityTypeAssociation": TestKnownToBeMissing,
-        "BidirectionalToManyEntityTypeAssociation": TestKnownToBeMissing,
-        "BidirectionalToOneEntityTypeAssociation": TestKnownToBeMissing,
+        "BidirectionalAssociation": TestKnownToBeMissing,
```

Comment:
Add comments that we test these base classes as pary.of the concrete `*To*` class tests.